### PR TITLE
SL-88 Create new encounter type for Outpatient Visit

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/SL-88

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/EncounterTypeBundle.java [hunk 1]
```diff
@@ -49,6 +49,8 @@ public void install() throws Exception {
         install(EncounterTypes.VACCINATION);
         install(EncounterTypes.ECHOCARDIOGRAM);
         install(EncounterTypes.MEXICO_CONSULT);
+        install(EncounterTypes.SIERRA_LEONE_OUTPATIENT_INITIAL);
+        install(EncounterTypes.SIERRA_LEONE_OUTPATIENT_FOLLOWUP);
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/EncounterTypeBundle.java]: I think it makes sense to start breaking these out into implementation-specific bundles... do you have a argument as to why not do to this?

Seems weird to have Sierra Leone encounter types in Haiti or Mexico, etc...

File: omod/src/main/webapp/resources/scripts/visit/encounterTypeConfig.js [hunk 2]
```diff
@@ -5,11 +5,6 @@ angular.module("encounterTypeConfig", ["constants"])
         var hfeSimpleEditUrl = "/htmlformentryui/htmlform/editHtmlFormWithSimpleUi.page?patientId={{encounter.patient.uuid}}&encounterId={{encounter.uuid}}&returnUrl={{returnUrl}}&breadcrumbOverride={{breadcrumbOverride}}";
         var hfeStandardEditUrl = "/htmlformentryui/htmlform/editHtmlFormWithStandardUi.page?patientId={{encounter.patient.uuid}}&encounterId={{encounter.uuid}}&returnUrl={{returnUrl}}&breadcrumbOverride={{breadcrumbOverride}}";
 
-        // if use has only "enterConsultNote", must be an active visit; retroConsultNote or retroConsultNoteThisProviderOnly required for retro visits
-        // TODO do we use this anywhere?
-        var standardConsultNoteRequire = "(user.hasPrivilege('Task: emr.enterConsultNote') && visit && !visit.stopDatetime) || (user.hasPrivilege('Task: emr.retroConsultNote') || user.hasPrivilege('Task: emr.retroConsultNoteThisProviderOnly'))"
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/scripts/visit/encounterTypeConfig.js]: Is there reason you removed this?  (Was it dangled code no longer being used?)

Comment (brandones) [reply omod/src/main/webapp/resources/scripts/visit/encounterTypeConfig.js]: ` // TODO do we use this anywhere?`... the answer appears to be no :)

File: api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/EncounterTypeBundle.java (hunk 1 above)
Comment (brandones) [reply api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/EncounterTypeBundle.java]: Yes, I think that is definitely the right path. Call it a near-future refactor?

**Comment by brandones** [comment]
Merging in order to un-block Martin.

**Pull request merged**